SDK/MobileCenterDistribute/Microsoft.Azure.Mobile.Distribute.iOS/Distribute.cs

```diff
@@ -41,5 +49,21 @@ public static void OpenUrl(NSUrl url)
         {
             iOSDistribute.OpenUrl(url);
         }
+
+        /// <summary>
+        /// Do not check for updates in case the app is launched with a debug configuration.
+        /// Place this method call into you app delegate's didFinishLaunching:withOptions: method BEFORE 
+        /// you call MobileCenter.start(...) if you are using in-app updates.
```

Comment:
before you call MobileCenter.Start or bebore you init the forms application object if you use Xamarin Forms.